dist/build.js

```diff
@@ -3059,9 +3041,6 @@ Branch.prototype.link = wrap(callback_params.CALLBACK_ERR_DATA, function(a, b) {
     a(null, f && f.url);
   });
 });
-Branch.prototype.sendSMS = function() {
-  console.warn("SMS feature has been deprecated. This is no-op.");
-};
```

Comment:
I want to note that removing this function impacts a product that I'm building. I'm releasing a hotfix for production.

Surely, it was my bad, as I didn't ensure it worked as expected. I should've checked the console warn and used a specific version instead of `latest`.

However, would you consider making it more foolproof? Maybe distributing a library via npm instead of `https://cdn.branch.io/branch-latest.min.js`? Then we can enable IntelliSense to show deprecated functions. Or something else.

I greatly appreciate all the work you have done. Just sharing some thoughts here.

Thank you!

Reply:
@jparksecurity we do distribute a npm package : https://www.npmjs.com/package/branch-sdk
please try that.

Reply:
Thank you! That's what I was looking for. I don't know why I haven't seen this till now.

Reply:
FYI I suggested the doc update. https://help.branch.io/developers-hub/suggested-edits/64f8ec3fcef95d006697f851